interfaces/cython/cantera/ck2yaml.py

```diff
@@ -32,8 +33,10 @@
 'surface'.
 
 The '--permissive' option allows certain recoverable parsing errors (e.g.
-duplicate transport data) to be ignored. The '--name=<name>' option
-is used to override default phase names (i.e. 'gas').
+duplicate transport data) to be ignored, whereas the '--name=<name>' option is
```

Comment:
I don't think "whereas" is the right word, here, it implies some comparison between the `--permissive` and `--name` options, which is not intended.

Reply:
I’ll go over this once the bigger picture clears up.

Reply:
Resolved.